Xamarin.Forms.Core/ToolbarItem.cs

```diff
@@ -12,11 +12,34 @@ public class ToolbarItem : MenuItem
 
 		static readonly BindableProperty PriorityProperty = BindableProperty.Create("Priority", typeof(int), typeof(ToolbarItem), 0);
 
+		public static readonly BindableProperty IsVisibleProperty = BindableProperty.Create("IsVisible", typeof(bool), typeof(ToolbarItem), true, BindingMode.TwoWay, propertyChanged: (bindable, oldvalue, newvalue) =>
```

Comment:
nameof(IsVisible)

Reply:
a few remarks:
- use `nameof()`,
- use the same binding mode as `VisualElement.IsVisibleProperty`,
- when the propertychanged handler is more than a single-line long, extract that to a method, for clarity

and then:
- for css-styling purposes, it'd be nice if the property was actually shared with VisualElement's. But as the ToolBarItems aren't CSS-selectable, I'm fine if we don't do it at this time